packages/sewing-kit-koa/src/tests/manifests-read.test.ts

```diff
@@ -85,6 +85,15 @@ describe('Manifests read', () => {
     expect(readJson).toHaveBeenCalledTimes(1);
   });
 
+  it('does not only reads manifests once when caching is disabled', async () => {
+    const manifests = new Manifests(null, false);
```

Comment:
Since `caching` is a new parameter and this is the only time parameters are passed to `Manifests` in this file, I think we'd benefit from naming them, even though it gets a bit verbose.

```suggestion
    const path = null;
    const caching = false; 
    const manifests = new Manifests(path, caching);
```

Also, I think this test's name could be improved. I imagine the idea was to directly negate the test above it, but I feel like it sounds a bit weird. What do you think about "reads manifests multiple times when caching is disabled"?